x-pack/plugins/security_solution/public/detections/components/host_isolation/isolate_sentinelone.tsx

```diff
@@ -0,0 +1,91 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { useMemo, useState, useCallback } from 'react';
+import { EuiSpacer } from '@elastic/eui';
+import { SENTINELONE_CONNECTOR_ID, SUB_ACTION } from '@kbn/stack-connectors-plugin/public/common';
+import { useLoadConnectors } from '../../../common/components/response_actions/use_load_connectors';
+import { useSubActionMutation } from '../../../timelines/components/side_panel/event_details/flyout/use_sub_action_mutation';
+import { RETURN_TO_ALERT_DETAILS } from './translations';
+import {
+  EndpointIsolateForm,
+  ActionCompletionReturnButton,
+} from '../../../common/components/endpoint/host_isolation';
+
+export const IsolateSentinelOneHost = React.memo(
```

Comment:
I'm not sure that the cypress tests cover all of the logic in this component (and `unisolate` component). Does it?

Reply:
I believe so

Reply:
Ok. One thing I did not see was a check to ensure the Description field is not displayed. Did you include that since my last review?